source/AndroidXProject.cshtml

```diff
@@ -151,7 +151,7 @@
     <!-- PackageReference -->
     @foreach (var dep in @Model.NuGetDependencies) {
       if (!dep.IsProjectReference) {
-      <PackageReference Include="@(dep.NuGetPackageId)" Version="@(dep.NuGetVersion)" />
+      <PackageReference Include="@(dep.NuGetPackageId)" Version="@(dep.NuGetVersion)" PrivateAssets="none" />

```

Comment:
This is to make sure that the .nupkg will correctly import the dependencies - such as Guava.